pkg/eval/builtin_fn_flow.go

```diff
@@ -50,7 +50,73 @@ import (
 
 // TODO(xiaq): Document "multi-error".
 
-// TODO(xiaq): Document "return", "break" and "continue".
+//elvdoc:fn break
+//
+// Raise the special "break" exception. Loops capture this exception and cause
+// the loop to terminate. Because it is an exception it can be caught by a
```

Comment:
> Raise the special "break" exception

Raise -> Raises. See https://github.com/elves/elvish/blob/master/CONTRIBUTING.md#reference-docs: The first sentence of the description should start with a verb in 3rd person singular (i.e. ending with a "s"), as if there is an implicit subject "this function".

> Loops capture this exception and cause the loop to terminate

The subject of this sentence is "loops", so the second half of this sentence is grammatically equivalent to "loops cause the loop to terminate". Here are some alternatives:

* Loops capture this exception, which causes the loop to terminate

* When raised inside a loop, it is captured by the loop and causes the loop to terminate

> Because it is an exception

Break the paragraph before this sentence: the first sentence alone is a complete specification of the behavior of `break`; the rest are comments.